### ISSUE TITLE
Support for GradleDoc

### ISSUE THREAD

**Issue by criztovyl** [StevenGBrown/javadoc-search-frame#15]
The left frame only shows "All Classes", but no search field.

Browser Console yells the folllowing:

```
13:55:47.771 TypeError: typeInHtml is undefined  Javadoc_Search_Frame.user.js:1173:1
```

Waterfox 55.2.2 (Firefox 56).

Thanks for this awesome userscript!

**Comment by criztovyl** [comment]
Reproduce:

http://docs.groovy-lang.org/latest/html/gapi/

![screen shot 2017-12-02 at 14 00 12](https://user-images.githubusercontent.com/2174918/33515892-26a378d4-d769-11e7-865e-8267aaf48021.png)

**Comment by StevenGBrown** [comment]
Fixed in 1.7.0
